registry/tracecat_registry/templates/detect/lookup_ip_address/ipinfoio.yml

```
@@ -0,0 +1,26 @@
+type: action
+definition:
+  name: lookup_ip_address
+  namespace: tools.ipinfoio
+  title: Lookup IP address
+  description: Get IPinfo.io report for an IP address
+  display_group: IPinfo.io
+  doc_url: https://ipinfo.io/developers
+  secrets:
+    - name: ipinfoio
+      keys:
+        - API_KEY
+  expects:
+    ip_address:
+      type: str
+      description: IP address to lookup.
+  steps:
+    - ref: getInformationByIp

```

```suggestion
    - ref: get_ip_info
```
Snakecase to keep with convention

---

Used the camelcase as it's defined in their openapi spec that way. Figured might be more in line for auto generating from api.